About.html

```diff
@@ -4,10 +4,135 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
     <title>Capstone1 About page</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
 </head>
 <body>
-    <script src="main.js"></script>
+    <!-- Header NAV BARS -->
+    <header class="header">
+        <!-- Only for Desktop -->
+         <div></div>
+        <div class="navbar1">
```

Comment:
- [ ] For semantic purposes, all your navigation menus should be enclosed in a `<nav>` tag. Please kindly fix that.